updater/developer_mode.php

```diff
@@ -83,6 +83,9 @@
 		$loader->output();
 	}
 	echo "</table>";
+	$loader->set_template_file("developer_mode_progress");
```

Comment:
We need a cool "LibreEHR Developer" logo thingie for dev mode, don'tcha think?  :)

Reply:
yes @aethelwulffe :relaxed: